source/appModules/notepad.py

```diff
@@ -41,6 +42,10 @@ def event_UIA_elementSelected(self, obj: NVDAObject, nextHandler: Callable[[], N
 				)
 			)
 		nextHandler()
+	
```

Comment:
We usually use a blank (empty) line between method definitions - in this line, it is a single tab with nothing after that.